lib/index.js

```diff
@@ -372,6 +398,27 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function (list) {
+  if (Array.isArray(list)) {
+    list.forEach(this.register.bind(this));
+  } else {
+    throw new TypeError('chan requires an Array');
+  }
+}
+
+Socket.prototype.rmchan     = function() {
+  var i = arguments.length;
+  while(i--) {
+    if (this.channels[arguments[i]]) {
+      if (nn.Chan(this.binding, nn.NN_SUB_UNSUBSCRIBE, arguments[i]) > -1) {
+        return delete this.channels[arguments[i]];
+      }
+      return this.emit('error', new Error( nn.Err() + ' : ' + chan));
+    }
+  };
+}
```

Comment:
Actually, the return values here are also pretty useless, let's remove explicit return statements and just have the innermost if be an if-else.